src/storage/storage.py

```diff
@@ -31,18 +37,24 @@ def __len__(self):
         raise NotImplementedError
 
     def by_type_and_tag(self, objectType, tag):
+        """"""
```

Comment:
`"""Return inventory vectors filtered by object type and tag"""`

Reply:
Okay will update, thanks!